AutoDarkModeApp/UserControls/ShortcutDialogContentControl.xaml.cs

```diff
@@ -0,0 +1,127 @@
+using Microsoft.UI.Input;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Windows.System;
+using Windows.UI.Core;
+
+namespace AutoDarkModeApp.UserControls;
+
+public sealed partial class ShortcutDialogContentControl : UserControl
+{
+    public List<SingleHotkeyDataObject> Keys
+    {
+        get => (List<SingleHotkeyDataObject>)GetValue(KeysProperty);
+        set => SetValue(KeysProperty, value);
+    }
+
+    public static readonly DependencyProperty KeysProperty = DependencyProperty.Register(
+        "Keys",
+        typeof(List<SingleHotkeyDataObject>),
+        typeof(ShortcutDialogContentControl),
+        new PropertyMetadata(default(string))
+    );
+
+    public string? CapturedHotkeys
+    {
+        get => (string?)GetValue(CapturedHotkeysProperty);
+        set => SetValue(CapturedHotkeysProperty, value);
+    }
+
+    public static readonly DependencyProperty CapturedHotkeysProperty = DependencyProperty.Register(
+        "CapturedHotkeys",
+        typeof(string),
+        typeof(ShortcutDialogContentControl),
+        new PropertyMetadata(default(string))
+    );
+
+    public ShortcutDialogContentControl()
+    {
+        InitializeComponent();
+    }
+
+    private void StackPanel_PreviewKeyDown(object sender, Microsoft.UI.Xaml.Input.KeyRoutedEventArgs e)
+    {
+        var key = e.Key;
+
+        var isCtrl = IsKeyDown(VirtualKey.Control);
+        var isShift = IsKeyDown(VirtualKey.Shift);
+        var isAlt = IsKeyDown(VirtualKey.Menu);
+        var isWin = IsKeyDown(VirtualKey.LeftWindows) || IsKeyDown(VirtualKey.RightWindows);
+        var clearTextBox = IsKeyDown(VirtualKey.Escape) || IsKeyDown(VirtualKey.Back) || IsKeyDown(VirtualKey.Delete);
+
+        var hotkey = "";
+        hotkey += isCtrl ? "Ctrl + " : "";
```

Comment:
As @Armin2208 and @Spiritreader can vouch, in some languages these names/abbreviations are different. In German "Ctrl" = "Strng" (for Steuerung)